ProcessMaker/processmaker
# FOUR-12174 | Front End: Develop Template Detail Slide

# Issue
Ticket: [FOUR-12174](https://processmaker.atlassian.net/browse/FOUR-12174)

This PR introduces the initial iteration of the Wizard Template Detail slide component as part of the ongoing development for the Wizard Templates feature. The Template Detail slide is designed to display comprehensive information about a selected Wizard Template.

# How to Test
1. Go to branch `task/FOUR-12174` in `processmaker`.
2. Go to the Processes Tab.
3. Select Wizard Templates under 'Add From Templates'.
4. Select one of the Wizard Template options.
5. A modal should appear with Wizard Template details.

ci:next

# Code Review Checklist
